src/BSeries.jl

```diff
@@ -1748,16 +1748,19 @@ function renormalize!(series)
 end
 
 """
-    energy_preserving_order(rk::RungeKuttaMethod, max_order)
+    energy_preserving_order(rk::RungeKuttaMethod, max_order; tol::Real=0)
```

Comment:
As I said, I would prefer using a default value that cannot be used as sensible value here - `nothing` instead of `0` to allow people to choose a vanishing tolerance.